### ISSUE TITLE
[Bug]: No skip for credits

### ISSUE THREAD

**Issue by provinzio** [jumoog/intro-skipper#178]
### Describe the bug

I freshly installed Jellyfin in a Proxmox LXC Container using tteck's scripts and the plugin (v0.2.0.6). The detection and skipping of intros seems to work fine, but I do not see any buttons to skip the credits.

The xml files for intro and credits have roughly the same size.
I checked one specific episode (https://www.imdb.com/title/tt1509989/?ref_=ttep_ep24) id which appears in both files and played it. It shows the skip intro button but no skip credit button.

On the plugin page :: manage fingerprints, I do not see credit timestamps.

For most of my testing, I used my Android Xiaomi 12 with the app Findroid. Using Firefox on my Laptop, I do not see any buttons (isn't the Firefox webplayer on the jellyfin webpage supported?).

### Jellyfin installation method

Proxmox LXC Container, tteck script: https://github.com/tteck/Proxmox/blob/main/install/jellyfin-install.sh

### Operating System

Ubuntu 22.04

### IMDb ID of that TV Series

tt1509989

### Support Bundle

```shell
* Jellyfin version: 10.9.2
* Plugin version: 0.2.0+46f280f0afae
* Queue contents: 311 episodes, 19 seasons
* Warnings: `UnableToAddSkipButton`
* FFmpeg: `okay`

FFmpeg version:

ffmpeg version 6.0.1-Jellyfin Copyright (c) 2000-2023 the FFmpeg developers
built with gcc 11 (Ubuntu 11.4.0-1ubuntu1~22.04)
configuration: --prefix=/usr/lib/jellyfin-ffmpeg --target-os=linux --extra-version=Jellyfin --disable-doc --disable-ffplay --disable-ptx-compression --disable-static --disable-libxcb --disable-sdl2 --disable-xlib --enable-lto --enable-gpl --enable-version3 --enable-shared --enable-gmp --enable-gnutls --enable-chromaprint --enable-opencl --enable-libdrm --enable-libass --enable-libfreetype --enable-libfribidi --enable-libfontconfig --enable-libbluray --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libopenmpt --enable-libdav1d --enable-libsvtav1 --enable-libwebp --enable-libvpx --enable-libx264 --enable-libx265 --enable-libzvbi --enable-libzimg --enable-libfdk-aac --arch=amd64 --enable-libshaderc --enable-libplacebo --enable-vulkan --enable-vaapi --enable-amf --enable-libvpl --enable-ffnvcodec --enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvdec --enable-nvenc
libavutil      58.  2.100 / 58.  2.100
libavcodec     60.  3.100 / 60.  3.100
libavformat    60.  3.100 / 60.  3.100
libavdevice    60.  1.100 / 60.  1.100
libavfilter     9.  3.100 /  9.  3.100
libswscale      7.  1.100 /  7.  1.100
libswresample   4. 10.100 /  4. 10.100
libpostproc    57.  1.100 / 57.  1.100
```
```


### Jellyfin logs

_No response_

**Comment by rlauuzo** [comment]
Currently, Findroid only supports a skip button for intros (see https://github.com/jarnedemeulemeester/findroid/pull/633). To enable the skip button functionality, you'll need to adjust the permissions of the index.html file.

**Comment by provinzio** [comment]
> To enable the skip button functionality in your browser, you'll need to adjust the permissions of the index.html file.

I saw an error in my log file and executed the suggested chown command, restarted the jellyfin container, but without any luck. 

The error is gone, but I still don't see a skip button in the browser.

**Comment by rlauuzo** [comment]
I'm not familiar with LXC containers, but if it's a standard Ubuntu setup and Jellyfin is running as user 'jellyfin', the command should likely be: 
`
chown jellyfin /usr/share/jellyfin/web/index.html
`
However, it's always a good idea to double-check the actual user ID that Jellyfin is running under in your LXC container to be sure. You can use the id jellyfin command within the container to confirm this.

**Comment by rlauuzo** [comment]
To use the skip intro button on the official Jellyfin Android app, you need to switch the video player type to "Web Player" in the app's client settings.

**Comment by provinzio** [comment]
Mhh, I thought I tried that multiple times... but for some reason, it's working now, yay.

I checked the android app which has the webplayer set as default. I restarted the app and I can see the buttons there now, too.

The androidtv app has no option for the webplayer, I'll check there repo for that.

Thanks for your quick response and support. :)


tl;dr Buttons are shown in the webplayer; Findroid does not support the skip-credits button currently (https://github.com/jarnedemeulemeester/findroid/pull/633); Jellyfin androidtv app does not support the webplayer right now.
